### DISCUSSION TITLE
Nginx as a Reverse Proxy or Jump server

### DISCUSSION THREAD

**Discussion by imtiazahmed93** [nginx-proxy/nginx-proxy#2332] [Q&A] [answered]
Hello Community,

I am deploying two nginx servers between APIGee and Data-source. APIgee is in Data-centerA and Data-source is in Data-centerB.

**DatacenterA**:  APIgee-A  <------> Nginx-A    **<=======>**   **Datacenter-B**:  Nginx-B <------> Datasource-B 

In the above scenario the api requests (Get, Response ) will be passing through two nginx servers, I reviewed some documentation and there is the Reverse Proxy i can use to establish the connectivity. But my Question is that can I use or how i will use reverse proxy on two nginx server, So that both or one of them act like a jump server for the API requests?

**Comment by SchoNie** [comment, marked as answer]
I would suggest asking this on the [Nginx community](https://forum.nginx.org/).
This project is about automatically generating nginx configs from Docker containers for nginx. (nginx-proxy ≠ nginx) Your question is about advanced use cases for Nginx and the experts are probably on their community website.

**Reply by imtiazahmed93** [reply]
Thank you SchoNie, I am posting  on the [Nginx community](https://forum.nginx.org/).
